ledger/sandbox/BUILD.bazel

```diff
@@ -250,6 +262,32 @@ client_server_test(
     ],
 ) if not is_windows else None
 
+client_server_test(
+    name = "conformance-test-static-time-h2database",
+    timeout = "long",
+    client = "//ledger/ledger-api-test-tool",
+    client_args = [
+        "--all-tests",
+        "--timeout-scale-factor=10",
+    ],
+    data = [
+        "//ledger/test-common:dar-files",
+    ],
+    server = "//ledger/sandbox:sandbox-ephemeral-h2database",
```

Comment:
This can just point to `//ledger/sandbox:sandbox-binary`, as the in-memory H2 jdbc URL by its very nature makes it an ephemeral H2 sandbox.

Reply:
👍